packages/react-core/src/demos/DataList/examples/DataListPagination.tsx

```diff
@@ -0,0 +1,187 @@
+        <TextContent>
+          <Text component="h1">Projects</Text>
+          <Text component="p">This is a demo that showcases PatternFly Data List</Text>
+        </TextContent>
+      </PageSection>
+      <PageSection isFilled>
+        <Toolbar id="toolbar-group-types">
+          <ToolbarContent>{toolbarItems}</ToolbarContent>
+        </Toolbar>
+        <DataList aria-label="Demo data list">
+          {paginatedRows.map((row, rowIndex) => {
+            const { name, threads, applications, workspaces, lastModified } = row;
+            return (
+              <DataListItem aria-labelledby={`Demo-item-${rowIndex}`} key={rowIndex}>
+                <DataListItemRow>
+                  <DataListItemCells
+                    dataListCells={[
+                      <DataListCell isFilled={false} key="buttons1">
+                        <Flex direction={{ default: 'column' }}>
+                          <FlexItem>
+                            <Text id={`Demo-item-${rowIndex}`} component={TextVariants.p}>
+                              {name}
+                            </Text>
+                          </FlexItem>
+                          <FlexItem>
+                            <Text component={TextVariants.small}>
+                              Working repo for
+                              <a href="http://www.patternfly.org/">PatternFly</a>
+                            </Text>
+                          </FlexItem>
+                          <FlexItem>
+                            <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+                              <FlexItem>
+                                <CodeBranchIcon /> {threads}
+                              </FlexItem>
+                              <FlexItem>
+                                <CodeIcon /> {applications}
+                              </FlexItem>
+                              <FlexItem>
+                                <CubeIcon /> {workspaces}
+                              </FlexItem>
+                              <FlexItem> Updated {lastModified}</FlexItem>
+                            </Flex>
+                          </FlexItem>
+                        </Flex>
+                      </DataListCell>,
+                      <DataListCell isFilled={false} alignRight key="secondary content align">
+                        <Flex>
+                          <FlexItem>
+                            <Button variant="secondary">Action</Button>
+                          </FlexItem>
+                          <FlexItem>
+                            <a href="#">Link</a>
+                          </FlexItem>
+                        </Flex>
+                      </DataListCell>
+                    ]}
+                  />
+                </DataListItemRow>
+              </DataListItem>
+            );
+          })}
+        </DataList>
+        {renderPagination('bottom-pagination', PaginationVariant.bottom, true, true)}
```

Comment:
From the issue title in https://github.com/patternfly/patternfly-react/issues/10247, if this is supposed to match the [HTML/core static bottom pagination demo](https://www.patternfly.org/components/data-list/html-demos/static-bottom-pagination/), you'll want to pass `isStatic` to this bottom pagination. By default bottom pagination is "sticky" on mobile, meaning it floats on the bottom of the viewport as you scroll. A static pagination won't float on mobile, and will always be at the very end of the datalist. You'll want to make this change, disabling `isSticky`

```suggestion
        {renderPagination('bottom-pagination', PaginationVariant.bottom, true, false)}
```

Then I'm guessing you might need another arg in `renderPagination()` to set `isStatic`

https://github.com/patternfly/patternfly-react/blob/1f492d118b31f73e347d17f5ee8911b34d79c89f/packages/react-core/src/demos/DataList/examples/DataListPagination.tsx#L63-L78